spec/unit/frameSpec.js

```diff
@@ -0,0 +1,45 @@
+describe("Frame", function () {
+  let frame;
+  
+  beforeEach(function () {
+    frame = new Frame();
+  });
+
+  describe("any new instances of thermostat class have empty rolls array", function () {
+    it("rolls array is empty for frame instance", function () {
+      expect(frame._rolls).toEqual([]);
+    });
+  });
+
+  describe("adds score after a bowl", function () {
+    it("adds score into rolls array", function () {
+      frame.addRoll(5);
+      expect(frame._rolls.length).toEqual(1);
+    });
+  });
+
+  describe("adds bonus score to bonus score variable", function () {
+    it("correctly updates bonus score", function () {
+      frame.addBonusScore(5);
+      expect(frame._bonus_score).toEqual(5);
+    });
+  });
+
+  describe("calculates the score for the frame", function () {
+    it("returns correct score for frame", function () {
+      frame.addRoll(5);
+      frame.addRoll(5);
+      frame.addBonusScore(5);
+      expect(frame.calcFrameTotal()).toEqual(15);
+    });
+  });
+
+  describe("updates instance variable to frame total", function () {
+    it("updates frame total to correct amount", function () {
+      frame.addRoll(5);
+      frame.addRoll(5);
+      frame.addBonusScore(5);
+      expect(frame.calcFrameTotal()).toEqual(15);
+    });
+  });
```

Comment:
These are the same test twice? It would be nice to see a few different frames in the tests here. Maybe a normal one, a spare and a strike.